src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random random = new Random();

```

Comment:
we use static for constants (which generally are primitive types), but not for reference-type fields
```suggestion
    private final Random random = new Random();
```